src/main/java/service/implement/ParceOutputValue.java

```diff
@@ -0,0 +1,18 @@
+package service.implement;
+
+import core.basesyntax.model.Fruit;
+import java.util.List;
+import service.ParceOutputValueService;
+
+public class ParceOutputValue implements ParceOutputValueService {
```

Comment:
parce - кажется, похоже
А вообще лучше назвать :
```suggestion
public class ReportServiceImpl implements ReportService {
```